### PR TITLE
Use let%server and let%client

### PR DESCRIPTION
**Pull request by dannywillems**

### CODE REVIEW

File: i18n_generate.mll [hunk 1]
```diff
@@ -91,22 +91,18 @@ let print_type fmt langs =
 let print_header fmt default_lang =
   Format.pp_print_string fmt @@
   "[%%shared let default_language = " ^ default_lang ^ "]\n\
```
Comment (sagotch) [inline i18n_generate.mll]: This one could be `let%shared`.

**Comment by dannywillems** [comment]
@sagotch Done.

File: i18n_generate.mll [hunk 2]
```diff
@@ -90,23 +90,19 @@ let print_type fmt langs =
...
+   let%server set_lang lang = Eliom_reference.Volatile.set _language_ lang\n\
+   let%client _language_ = ref default_language\n\
+   let%client get_lang () = !_language_\n\
+   let%client set_lang lang = _language_ := lang\n\
    \n\
-   [%%shared\n\
    [@@@ocaml.warning \"-27\"]\n\
-   let pcdata = Eliom_content.Html.F.pcdata\n\
+   let%shared pcdata = Eliom_content.Html.F.pcdata\n\
+   [%%shared\n\
```
Comment (sagotch) [inline i18n_generate.mll]: I just realized that we style use the `[%%shared ... ]` syntax here, which is a little bit strange regarding this pull request...

Comment (dannywillems) [inline i18n_generate.mll]: @sagotch Because it's for the module language.

Comment (sagotch) [inline i18n_generate.mll]: Right...

Comment (sagotch) [inline i18n_generate.mll]: (I just released ocsigen-i18n without your pull requests because I did not take time to test it yet...)

**Comment by dannywillems** [comment]
@sagotch I rebased.

**Comment by sagotch** [comment]
Included in another PR, thanks.

**Pull request closed without merging**